core/metacling/src/TClingCallFunc.cxx

```diff
@@ -1187,6 +1190,17 @@ tcling_callfunc_ctor_Wrapper_t TClingCallFunc::make_ctor_wrapper(const TClingCla
    buf << wrapper_name;
    buf << "(void** ret, void* arena, unsigned long nary)\n";
    buf << "{\n";
+   ++indent_level;
+
+   string constr_arg;
+   if (kind == ROOT::TMetaUtils::EIOCtorCategory::kIOPtrType)
+      constr_arg = "((TRootIOCtor*)nullptr)";
+   else if (kind == ROOT::TMetaUtils::EIOCtorCategory::kIORefType) {
+      indent(buf, indent_level);
+      buf << "TRootIOCtor arg;\n";
```

Comment:
TRootIOCtor is (intentionally) never defined, just forward declared.

Reply:
It is defined - here: https://github.com/root-project/root/blob/master/core/base/inc/TRootIOCtor.h

Reply:
Still ... for consistency with the rest of the advise regarding TRootIOCtor just use
```
       buf << "TRootIOCtor* iomarker = nullptr;\n";
```

Reply:
I modify these lines and just using `(*((TRootIOCtor*)nullptr))`